UCDlist.tex

```diff
@@ -816,8 +816,18 @@ \subsection{P or S syntax code: Which is the most pertinent position for a UCD w
 Concatenation can apply more than one time, depending on ordering rules. See \citet{2005ivoa.spec.0819D}, section 3.3, 
 for other details.
 
-
 \section{Changes from previous versions}
+\subsection{Changes from UCDList EN v1.4 following RFM}
+\subsubsection*{Additions}
+\emph{stat.skewness}, \emph{stat.kurtosis},\emph{stat.mad} \\
+\emph{stat.percentile} \\
+\emph{meta.ref.pid}, \emph{meta.ref.epic}, \emph{meta.ref.rorid}, \emph{meta.ref.orcid} \\
+
+\subsubsection*{Ammendment}
+\emph{meta.curation} changed to Q to allow combination as secondary term. The former usage \emph{meta.curation} for the id or name of a curator is discouraged. \emph{meta.id;metacuration} is recommended instead.
```

Comment:
Is this supposed to read "meta.id;meta.curation"?

Reply:
yes sure . thanks Mark for spotting this.